SofaKernel/extlibs/newmat/CMakeLists.txt

```diff
@@ -41,7 +41,12 @@ set(SOURCE_FILES
 add_library(${PROJECT_NAME} STATIC ${HEADER_FILES} ${SOURCE_FILES})
 target_include_directories(${PROJECT_NAME} PUBLIC "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>")
 target_include_directories(${PROJECT_NAME} PUBLIC "$<INSTALL_INTERFACE:include>")
-target_compile_definitions(${PROJECT_NAME} PRIVATE "use_namespace")
+target_compile_definitions(${PROJECT_NAME} PUBLIC "use_namespace")
+if(NOT DEFINED SOFA_WITH_DOUBLE)
```

Comment:
To be fully clean, it would be great to add this code where the newmat dir in included in cmake.
The idea is that nothing would be required when updating newmat.